packages/web-react/src/components/UNSTABLE_Truncate/stories/UNSTABLE_Truncate.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import ReadMe from '../README.md';
+import { UNSTABLE_Truncate } from '..';
```

Comment:
⚠️ I know that this not part of the production package, but it is more safe to use file import.

```suggestion
import UNSTABLE_Truncate from '../UNSTABLE_Truncate';
```

See https://github.com/lmc-eu/spirit-design-system/pull/1464.